src/glide/docs/docs.js

```diff
@@ -123,6 +123,174 @@
     return window.pagefind_ui;
   };
 
+  /**
+   * TOC configuration per page
+   * Maps page filename to CSS selector for headings to include.
+   * Only headings with `id` attributes are included.
+   *
+   * @example
+   * "api.html": "h1[id], h2[id]"
+   * "keys.html": "h1[id], h2[id], h3[id]"
+   */
+  const TOC_CONFIG = {
```

Comment:
I think we should only add this to pages that are listing a bunch of things, not prose content.

So that would result in just
```typescript
    "api.html": "h1[id], h2[id]",
    "excmds.html": "h1[id], h2[id]",
    "changelog.html": "h1[id]",
    "autocmds.html": "h1[id], h2[id]",
```